mnemonic/mnemonic.py

```diff
@@ -250,15 +250,18 @@ def to_seed(cls, mnemonic, passphrase=""):
         return stretched[:64]
 
     @classmethod
-    def to_hd_master_key(cls, seed):
+    def to_hd_master_key(cls, seed, network='main'):
```

Comment:
`def to_hd_master_key(cls, seed, testnet=False):`

Reply:
@prusnak Thank you for your review.
I modified.